data_structures/linked_list/ascendingPriorityQueue.c

```diff
@@ -0,0 +1,169 @@
+/* Ascending priority queue using Linked List - Program to implement Ascending priority queue using Linked List */
+
```

Comment:
@shubhamsah Great work, kindly add a sample case of your program with the state of the queue in each operation, like m insert() followed by n delete() and exit() fundtions

Reply:
Thanks for the review, But could you specify an example so that I can get better understanding of what changes I need to make.

Reply:
@shubhamsah make some larger insert() and delete() calls to show the priority.
input        state of queue
1 
2                       2
1
3                        2->3
2                        3
3

Output
2

Reply:
Roger... Will do it

Reply:
Thank you for the review, 
I am still not able to understand the context of the review. Do you mean to add a function to test/run my code or should I comment out the output? Sorry to disturb you but even if you could give a previous file with a similar algorithm and the sample included I can learn and mold my file accordingly as I can't see a file that includes one such. And if there aren't any changes then I guess we are good to merge.

Reply:
My fault there is a indentation problem, just comment out the output . I have shown some, but use a larger number of insert().

Reply:
Yes I got it... but there's an error in your test case since this is an ascending priority queue so the smallest number in the queue would be removed
so the final output would be 3